src/python/pants/core/util_rules/environments.py

```diff
@@ -222,10 +240,13 @@ class DockerEnvironmentTarget(Target):
     )
     help = softwrap(
         """
-        Configuration of a Docker image used for building your code, including the environment
-        variables and search paths used by Pants.
+        Configuration of a Docker environment used for building your code, including the Docker
+        image, along with environment-sensitive options, which include environment variables and
+        search paths, used by Pants to execute processes in this Docker environment.
```

Comment:
This sentence has too many commas in it I think: not sure how to fix that. Break it up until multiple sentences, or make part of it parenthetical maybe?